### PR TITLE
first test cypress

### PR DESCRIPTION
**Pull request by Ktagudelo**

### CODE REVIEW

**Comment by Ktagudelo** [comment]
image

File: .prettierrc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-	"semi": false,
+	"semi": true,
```
Comment (danielgalvis98) [inline .prettierrc:2]: Why this change?

Comment (kliver98) [reply .prettierrc:2]: And why do you have this file if Prettier is not installed in the project and not mentioned in the workshop either? (package.json)

Comment (Ktagudelo) [reply .prettierrc:2]: Hi,
semi: true is equal to ; in the end
semi: false is without ;

Because I have prettier installed in my vscode and it conflicts with eslinc. No problem i will remove the file and uninstall prettier.

**Review by holgiosalos — APPROVED** [review]
